assets/css/style.css

```diff
@@ -33,7 +33,8 @@ footer{
     display: flex;
     align-items: center;
     gap: .5rem;
-    padding: 0rem 1rem;
+    padding: 0rem 2rem;
+    box-sizing: border-box;
```

Comment:
What does this particularly change with this element?

Reply:
by adding the box-sizing i wanted the padding not to effect the box size